public/views/goggles.html

```diff
@@ -0,0 +1,25 @@
+<div class="goggles-backgrounded goggles-xray-image">
+
+      <img src="/Shapes.jpg">
+
+      <div class="goggles-main-box goggles-box">
+        <h1>Remix this page</h1>
+        <ol class="goggles-listing">
+          <li><p><button class="goggles-activate-button">Activate X-ray Goggles</button></p></li>
+          <li><p>Tap the background image</p></li>
+          <li><p>Change the image's <i>source</i> from Shapes.jpg to Blur.jpg
```

Comment:
that would seem odd: this is not a quote, it's quite literally an eliding apostrophe, so ' is the correct symbol, with the font we're using making sure it's styled the way we want. Are we using quotation marks on other page? O_o